### PR TITLE
Readme update

### PR DESCRIPTION
**Pull request by ljacomet**
* Recommend RVM instead of bundler
* Provide gem list in rvm import format
* Disable hardbreaks in asciidoctor content

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,33 +4,23 @@ This is the source code/files for the ehcache.org website.  The files in this re
...
-
+  * Create a branch to contribute content
 * To generate the *full* site including Ehcache 3 docs, you need to do the following:
-  * within the root your clone of this repository, the contents of ehcache3 repository needs to be linked/or copied as "_eh3"
-  * within the root your clone of this repository, the contents of ehcache3 repository's docs/src/docs/asciidoc/user folder needs to be linked/or copied as "documentation/3.0"
-
-* To generate and view (locally serve) the site "bundle exec jekyll serve -w"   ( then point your browser at http://localhost:4000" )
-
-* To simply generate the site "jekyll build"
+  * within the root your clone of this repository, the contents of ehcache3 repository needs to be linked/or copied as `_eh3`
```
Comment (albinsuresh) [inline README.md]: within the root your clone of this repository -> from within the root **of** your clone of this repository

File: README.md [hunk 2]
```diff
@@ -4,33 +4,23 @@ This is the source code/files for the ehcache.org website.  The files in this re
...
+  * within the root your clone of this repository, the contents of ehcache3 repository's docs/src/docs/asciidoc/user folder needs to be linked/or copied as `documentation/3.0`
+* To generate and view (locally serve) the site `jekyll serve` then point your browser at `http://localhost:4000`
```
Comment (albinsuresh) [inline README.md]: How about adding a minor note on that double symlink error? That it can be ignored?

Comment (mathieucarbou) [inline README.md]: I was wondering too what is this error and where it comes from ?

Comment (ljacomet) [inline README.md]: Good point - the error comes from the fact that jekyll ends up trying to watch twice the same folder.

**Comment by ljacomet** [comment]
Updated the PR following the comments

**Pull request merged**